src/tracks/playabletrack/wavetrack/ui/WaveChannelVZoomHandle.cpp

```diff
@@ -125,34 +117,25 @@ UIHandle::Result WaveChannelVZoomHandle::DoRelease(
       return data.result;
    }
    else {
-      bool bVZoom;
-      gPrefs->Read(wxT("/GUI/VerticalZooming"), &bVZoom, false);
       // Ignore Capture Lost event 
-      bVZoom &= event.GetId() != kCaptureLostEventId;
-      // shiftDown | rightUp | ZoomKind
-      //    T      |    T    | 1to1
-      //    T      |    F    | Out
-      //    F      |    -    | In
-      if( bVZoom ) {
-         if( shiftDown )
-            zoomStart = zoomEnd;
-         doZoom(pProject, pTrack,
-            shiftDown
-               ? (rightUp ? kZoom1to1 : kZoomOut)
-               : kZoomIn,
-            rect, zoomStart, zoomEnd, !shiftDown);
-      }
+       bool notLost = event.GetId() != kCaptureLostEventId;
```

Comment:
Indentation.  The convention is three spaces in most of our source files.  You might change preferences in your source code editor.